README.md

```diff
@@ -133,11 +133,13 @@ For instance
 
 You'll need to install the dev requirements to run the tests:
 
-    pip install -r dev-requirements.txt
+To run the tests on CKAN >= 2.9, do:
```

Comment:
Sorry, that was a mistake on the ckanext-dcat README, I fixed now (https://github.com/ckan/ckanext-dcat/commit/c4e7319aff55e5315831b28e918299092f2d5805). We should use pytest across all CKAN versions, nose should not be used at all any more.